### ISSUE TITLE
[types] After update to 4.4: Enum declarations can only merge with namespace or other enum declarations

### ISSUE THREAD

**Issue by stefanmeschke** [supasate/connected-react-router#116]
Hi,
after 4.4.0 upgrade I run into the following error:

### Error
```
node_modules/connected-react-router/index.d.ts
(10,15): Enum declarations can only merge with namespace or other enum declarations.
```

### Proposal
I think the `index.d.ts` should be changed, because the enum on line 10 ([enum RouterAction](https://github.com/supasate/connected-react-router/blob/c6411d31b30fbcc9482aaa423a47434173270cc3/index.d.ts#L10)) and the interface on line 44 ([interface RouterAction](https://github.com/supasate/connected-react-router/blob/c6411d31b30fbcc9482aaa423a47434173270cc3/index.d.ts#L44)) have the same name.

Should I add a PR for that? :)

Cheers

**Comment by conrad** [comment]
This is blocking me as well. Unfortunately, I don't seem to have permission to push to a branch to be able to create a PR...

**Comment by supasate** [comment]
Released the fix in v4.4.1. Thanks @Armour for the quick fix.
